utils/log.py

```diff
@@ -3,14 +3,15 @@
 
 def get_simple_debug_logger(name):
     logger = logging.getLogger(name)
-    original_debug = logger.debug
+    original_makeRecord = logger.makeRecord
```

Comment:
this change was because overwriting `debug` also overwrites the tracing of the function and line number of the log statement